admin/routes/check-form.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+const isAuthenticated = require('../authentication/middleware')
+const rolesConfig = require('../roles-config')
+const checkFormV2Controller = require('../controllers/check-form-v2')
+const featureToggles = require('feature-toggles')
+
```

Comment:
as jon mentioned - why do we need new URLs.  the toggle can exist at the routing level to determine which implementation should be used, with no change on the app surface.